content/docs/ops/runbook/rotating-secrets.md

```diff
@@ -0,0 +1,187 @@
+---
+menu:
+  docs:
+    parent: runbook
+
+title: Rotating Secrets
+---
+
+This guide assumes that you're following the Offboarding Checklist [found
+here](https://github.com/18F/cg-product/blob/master/OffboardingChecklist.md) and
+removed direct access to cloud.gov AWS accounts. The following guide walks a
+cloud.gov operator through rotating all sensitive information (secrets)
+including the following:
+
+- Passphrases
+    - [Read about secret key management]({{< relref "docs/ops/secrets.md" >}})
+- AWS IAM users
+    - [Read about rotating IAM users]({{< relref "docs/ops/runbook/rotating-iam-users.md" >}})
+- Deployer users
+- Certificates
+- Clients
+- Service Brokers
+
+### Start by rotating access to various services and IAM accounts.
+
+- Delete any IAM keys and users from AWS consoles belonging to the user.
+- [Rotate all IAM Users]({{< relref "docs/ops/runbook/rotating-iam-users.md" >}})
+- [Rotating Concourse secrets]({{< relref "docs/ops/runbook/rotating-concourse.md" >}})
+- [Rotating Bosh secrets]({{< relref "docs/ops/runbook/rotating-bosh.md" >}})
+- [Rotating Cloud Foundry / Diego secrets]({{< relref "docs/ops/runbook/rotating-cloudfoundry-diego.md" >}})
+
+### Rotate CF deployment users
+
+Some of our Concourse pipelines deploy to Cloud Foundry using cloud.gov service
+account deployers. In order to rotate these, you need to target the correct
+org and space in order to create the cloud.gov service deployer account
+instance.
+
+#### Auditing all CF deployment users
+
+Use the `get-all-cf-deployers.sh` file in the
+[`cg-scripts`](https://github.com/18F/cg-scripts) repository to audit all of the
+CF users deploying to all environments using Concourse.
+
+The script outputs the login commands for all matching environments for both the
+deployer account and for operators using `--sso`. It's useful to run this
+command redirect the output to a file for the next steps.
+
+When rotating these accounts, you need ensure you don't delete the previous
+deployer service instance _until after_ the credentials have been successfully
+used in a deployment in order to minimize downtime.
+
+#### Steps for rotating cloud.gov Service Account deployers
+
+1. Target the appropriate org and space in the current environment.
+1. Check the services, if one exists for this deployer renamed it with a `_prev`
+   suffix for later deletion.
+   - `cf rename-service ${name_of_deployer} ${name_of_deployer}_prev`
+1. Create a new service instance.
+  - `cf create-service cloud-gov-service-account space-deployer "${name_of_deployer}"`
+1. Read the service instance details for the deployer.
+  - `cf service "${name_of_deployer}"`
+1. Open the Fugacious link and save the credentials locally in the `credentials.yml`
+  - `cf service ${name_of_deployer} | grep Dashboard | awk '{ print $2 }'`
+1. If the deployer user is used for more than a `cf push`, give it the proper
```

Comment:
yeah good point. are we worried at all about have those users extending the CF deployment? I'm thinking since most of these deployers exist in the same `org` and `space` we would only need a handful of the ones we currently have deployed.